bigtable: TestIntegration_Pinger failed

This test failed!

To configure my behavior, see [the Flaky Bot documentation](https://github.com/googleapis/repo-automation-bots/tree/main/packages/flakybot).

If I'm commenting on this issue too often, add the `flakybot: quiet` label and
I will stop commenting.

---

commit: 2f3c91847f882cef939496b92d5c4311583b4f5e
buildURL: [Build Status](https://source.cloud.google.com/results/invocations/e66b5843-4b1c-47b1-a734-5ac02521846b), [Sponge](http://sponge2/e66b5843-4b1c-47b1-a734-5ac02521846b)
status: failed

Error from logs:

```go
=== RUN   TestIntegration_Pinger
=== PAUSE TestIntegration_Pinger
=== CONT  TestIntegration_Pinger
    integration_test.go:233: pinger failed. got rpc error: code = FailedPrecondition desc = Table currently being created: projects/dulcet-port-762/instances/gc-bt-it-instance/tables/mytable-1780173133554513527-01, want <nil>
--- FAIL: TestIntegration_Pinger (0.06s)
```

--- a/bigtable/integration_test.go
+++ b/bigtable/integration_test.go
@@ -229,11 +229,26 @@ func TestIntegration_Pinger(t *testing.T) {
 	if !testEnv.Config().UseProd {
 		t.Skip("emulator doesn't support PingAndWarm")
 	}
-	if err := client.PingAndWarm(ctx); err != nil {
+	err = internal.Retry(ctx, gax.Backoff{
+		Initial:    2 * time.Second,
+		Max:        30 * time.Second,
+		Multiplier: 2.0,
+	}, func() (bool, error) {
+		err := client.PingAndWarm(ctx)
+		if err != nil {
+			s, ok := status.FromError(err)
+			if ok && s.Code() == codes.FailedPrecondition {
+				return false, err // retry
+			}
+			return true, err // stop and return other errors
+		}
+		return true, nil // success
+	})
+	if err != nil {
 		t.Fatalf("pinger failed. got %v, want %v", err, nil)
 	}
-
 }
+
 
 func TestIntegration_UpdateFamilyValueType(t *testing.T) {
 	t.Parallel()

--- a/bigtable/integration_test.go
+++ b/bigtable/integration_test.go
@@ -249,7 +249,6 @@ func TestIntegration_Pinger(t *testing.T) {
 	}
 }
 
-
 func TestIntegration_UpdateFamilyValueType(t *testing.T) {
 	t.Parallel()
 	ctx := context.Background()

